packages/cashier/src/pages/payment-agent/payment-agent-unlisted-withdraw-form/payment-agent-unlisted-withdraw-form.tsx

```diff
@@ -36,23 +51,28 @@ const validateWithdrawal = (values, { balance, currency }) => {
     return errors;
 };
 
-const PaymentAgentUnlistedWithdrawForm = observer(({ verification_code, setIsUnlistedWithdraw }) => {
+const PaymentAgentUnlistedWithdrawForm = observer(({ setIsUnlistedWithdraw }: TPaymentAgentUnlistedWithdrawForm) => {
     const { client } = useStore();
     const { balance, currency } = client;
+    const verification_code = client.verification_code.payment_agent_withdraw;
     const { payment_agent } = useCashierStore();
     const { error, onMountPaymentAgentWithdraw: onMount, requestTryPaymentAgentWithdraw } = payment_agent;
 
     React.useEffect(() => {
         onMount();
     }, [onMount]);
 
-    const validateWithdrawalPassthrough = values => validateWithdrawal(values, { balance, currency });
+    const validateWithdrawalPassthrough = (values: TValidateWithdrawalValueProps) =>
+        validateWithdrawal(values, { balance, currency });
 
-    const onWithdrawalPassthrough = async (values, actions) => {
+    const onWithdrawalPassthrough = async (
+        values: TValidateWithdrawalValueProps,
+        actions: { setSubmitting: (value: boolean) => void }
+    ) => {
         const payment_agent_withdraw = await requestTryPaymentAgentWithdraw({
             loginid: values.account_number,
             currency,
-            amount: values.amount,
+            amount: Number(values.amount),
             verification_code,
         });
         if (payment_agent_withdraw?.error) {
```

Comment:
Shall I remove this check, considering that:
1. `const payment_agent_withdraw = await requestTryPaymentAgentWithdraw({ ...`
2. `requestTryPaymentAgentWithdraw` doesn't return anything https://github.com/rostislav-deriv/deriv-app/blob/rostislav/84941/ts-migration-cashier/packages/cashier/src/stores/payment-agent-store.ts#L348-L374
🤔

Reply:
Same in `payment-agent-listed-withdraw-form`